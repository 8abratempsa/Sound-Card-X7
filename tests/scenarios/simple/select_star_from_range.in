octosql "SELECT * FROM range(start => 1, end => 10) r" --output stream_native
