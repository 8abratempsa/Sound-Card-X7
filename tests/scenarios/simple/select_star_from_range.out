{+0001-01-01T00:00:00Z| 1 |}
{+0001-01-01T00:00:00Z| 2 |}
{+0001-01-01T00:00:00Z| 3 |}
{+0001-01-01T00:00:00Z| 4 |}
{+0001-01-01T00:00:00Z| 5 |}
{+0001-01-01T00:00:00Z| 6 |}
{+0001-01-01T00:00:00Z| 7 |}
{+0001-01-01T00:00:00Z| 8 |}
{+0001-01-01T00:00:00Z| 9 |}
